pkg/local_object_storage/metabase/inhume.go

```diff
@@ -164,10 +186,12 @@ func (db *DB) Inhume(prm InhumePrm) (res InhumeRes, err error) {
 			obj, err := db.get(tx, prm.target[i], buf, false, true, currEpoch)
 			targetKey := addressKey(prm.target[i], buf)
 			if err == nil {
+				containerID, _ := obj.ContainerID()
 				if inGraveyardWithKey(targetKey, graveyardBKT, garbageBKT) == 0 {
-					// object is available, decrement the
+					// object is available, increment the
 					// logical counter
 					inhumed++
```

Comment:
Can we remove this comment? It is (was) misleading `logical counter` shouldn't increase at all on removal, what we increase is some other counter.